packages/angular-material/test/table-control.spec.ts

```diff
@@ -0,0 +1,295 @@
+      // 2 columns
+      expect(fixture.nativeElement.querySelectorAll('th').length).toBe(2);
+      // 1 head column and 2 data columns
+      expect(fixture.nativeElement.querySelectorAll('tr').length).toBe(1 + 2);
+      // 4 data entries
+      expect(fixture.nativeElement.querySelectorAll('td').length).toBe(4);
+    });
+  }));
+  it('renders object array on path', async(() => {
+    const mockSubStore = MockNgRedux.getSelectorStub();
+    component.uischema = uischema2;
+    component.schema = schema_object2;
+
+    mockSubStore.next({
+      jsonforms: {
+        renderers: renderers,
+        core: {
+          data: {
+            my: [{ foo: 'foo_1', bar: 'bar_1' }, { foo: 'foo_2', bar: 'bar_2' }]
+          },
+          schema: schema_object2
+        }
+      }
+    });
+    mockSubStore.complete();
+    fixture.detectChanges();
+    component.ngOnInit();
+    fixture.whenStable().then(() => {
+      // 2 columns
+      expect(fixture.nativeElement.querySelectorAll('th').length).toBe(2);
+      // 1 head column and 2 data columns
+      expect(fixture.nativeElement.querySelectorAll('tr').length).toBe(1 + 2);
+      // 4 data entries
+      expect(fixture.nativeElement.querySelectorAll('td').length).toBe(4);
+    });
+  }));
+
+  it('renders simple array on root', async(() => {
+    const mockSubStore = MockNgRedux.getSelectorStub();
+    component.uischema = uischema1;
+    component.schema = schema_simple1;
+
+    mockSubStore.next({
+      jsonforms: {
+        renderers: renderers,
+        core: {
+          data: ['foo', 'bar'],
+          schema: schema_simple1
+        }
+      }
+    });
+    mockSubStore.complete();
+    fixture.detectChanges();
+    component.ngOnInit();
+    fixture.whenStable().then(() => {
+      // 1 column
+      expect(fixture.nativeElement.querySelectorAll('th').length).toBe(1);
+      // 1 head column and 2 data columns
+      expect(fixture.nativeElement.querySelectorAll('tr').length).toBe(1 + 2);
+      // 2 data entries
+      expect(fixture.nativeElement.querySelectorAll('td').length).toBe(2);
+    });
+  }));
+  it('renders simple array on path', async(() => {
+    const mockSubStore = MockNgRedux.getSelectorStub();
+    component.uischema = uischema2;
+    component.schema = schema_simple2;
+
+    mockSubStore.next({
+      jsonforms: {
+        renderers: renderers,
+        core: {
+          data: { my: ['foo', 'bar'] },
+          schema: schema_simple2
+        }
+      }
+    });
+    mockSubStore.complete();
+    fixture.detectChanges();
+    component.ngOnInit();
+    fixture.whenStable().then(() => {
+      // 1 columns
+      expect(fixture.nativeElement.querySelectorAll('th').length).toBe(1);
+      // 1 head column and 2 data columns
+      expect(fixture.nativeElement.querySelectorAll('tr').length).toBe(1 + 2);
+      // 2 data entries
+      expect(fixture.nativeElement.querySelectorAll('td').length).toBe(2);
+    });
+  }));
+
+  xit('can be disabled', async(() => {
```

Comment:
Are these two tests disabled on purpose?

Reply:
I activated the enabled by default test case, the disabled one could be enabled after we merge #1371

Reply:
#1371 has been merged, we could update now.